tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/TestContexts/UsersControllerTestContext.cs

```diff
@@ -32,6 +34,8 @@ private UsersControllerTestContext()
 
         internal Mock<IRegistrationService> RegistrationServiceMock { get; set; }
 
+        public ApplicationUser GetUserById { get; set; }
```

Comment:
I think this might be better just named `User` as the current name is slightly confusing at first glance (and it has the potential for later reuse).